protoc-gen-insomniaenv/main.go

```diff
@@ -97,9 +98,10 @@ func (e *insomniaenv) Generate(in *plugin.CodeGeneratorRequest) (*plugin.CodeGen
 		return nil, err
 	}
 
-	e.registry = typemap.New(filesToGenerate)
+	e.registry = typemap.New(in.ProtoFile)
```

Comment:
This ensures that files which do not contain a twirp service but do have messages which are imported in twirp proto files get their message types added to the registry